tests/positive/files/file.go

```diff
@@ -65,6 +65,40 @@ func CreateFileOnRoot() types.Test {
 	}
 }
 
+func CreateFileOnRootFromBase64() types.Test {
+	name := "Create Files on the Root Filesystem from base64 data"
+	in := types.GetBaseDisk()
+	out := types.GetBaseDisk()
+	config := `{
+	  "ignition": { "version": "$version" },
+	  "storage": {
+	    "files": [{
+	      "filesystem": "root",
```

Comment:
This is spec 2.x syntax.  :slightly_smiling_face:

Reply:
Done